Thesis.tex

```diff
@@ -171,6 +171,26 @@ \section{Mentality concerning IP rights}
 
 \section{Globalisation \& increased complexity of rights management}
 \label{sec:GlobalRightsMgmt}
+The global development of industries over the past few decades had quite a significant impact related to patents. More companies from all over the world hold patents relevant to sometimes whole industry branches. The number of patents involved with modern products and services also reached a new scale. One example for this is the LTE \footcite{Long Term Evolution} standard for mobile communications that lead to several law suits between different technology companies over a number of years.
```

Comment:
<<<< Differing opinions and a multitude of affected patents belonging to different technology **companies led** to several law suits between these parties and a delay of years until the adaption of this standard.